modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -5,11 +5,7 @@
 [id="nw-ovn-kubernetes-limitations_{context}"]
 = OVN-Kubernetes limitations
 
-The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has the following limitations:
-
-* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`.
-The default value, `cluster`, is supported for both parameters.
-This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
```

Comment:
We should refactor this part to say this limitation can affect you when you add a service of type `ClusterIP`, `LoadBalancer`, `NodePort` or add a service with externalIP, since ITP can be done on a clusterIP as well.

Reply:
Wow, thank you @tssurya.  I revised to bring the bullet back, but remote ETP and ack that ClusterIP is affected too.